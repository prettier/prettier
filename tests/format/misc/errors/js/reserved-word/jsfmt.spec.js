run_spec(
  {
    dirname: __dirname,
    snippets: ["class interface {}", 'import interface from "foo";'],
  },
  ["espree", "meriyah"]
);
