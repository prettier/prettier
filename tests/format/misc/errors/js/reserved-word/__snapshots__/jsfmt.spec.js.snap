// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snippet: #0 [espree] format 1`] = `
"The keyword 'interface' is reserved (1:7)
> 1 | class interface {}
    |       ^"
`;

exports[`snippet: #0 [meriyah] format 1`] = `
"[1:15]: Unexpected strict mode reserved word (1:15)
> 1 | class interface {}
    |               ^"
`;

exports[`snippet: #1 [espree] format 1`] = `
"The keyword 'interface' is reserved (1:8)
> 1 | import interface from \\"foo\\";
    |        ^"
`;

exports[`snippet: #1 [meriyah] format 1`] = `
"[1:16]: Unexpected strict mode reserved word (1:16)
> 1 | import interface from \\"foo\\";
    |                ^"
`;
